features/withdrawing-a-manual.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Script to withdraw published manuals
+  As a DevOps specialist
+  I want to withdraw a manual
+  So that it is not accessible to the public
+
+  Scenario:
+    Given I am logged in as a "CMA" editor
+    And a published manual with at least two sections exists
+    When I withdraw the manual
```

Comment:
I think the name of this step is a bit misleading - the manual is actually being withdrawn by the DevOps specialist, not the CMA editor, and users of the app can't do that yet. If we add that functionality later, we may end up accidentally using this step when we intend to test something different, so I think it's worth being more accurate here to avoid future confusion.